src/components/menu/menu.component.jsx

```diff
@@ -36,7 +36,7 @@ function Menu({ setIsNewStory }) {
         isActive={activeBtn === "past"}
         handleBtnClick={handleActiveBtn}
       >
-        Past
+        Best
```

Comment:
This is fine!

Reply:
@Harsh97x raise a new PR only with the above change, and no changes in package.json file.